doc/Learn/Navigation/Walkthrough/Navigate Using Attached Properties in XAML.md

```diff
@@ -0,0 +1,232 @@
+---
+uid: Uno.Extensions.Navigation.Walkthrough.NavigateXAML
+title: Navigate Between Pages using Navigation Attached Properties in XAML
+summary: How to navigate declaratively from XAML using Navigation.Request and Navigation.Data attached properties without code-behind.
+tags: [uno, uno-platform, uno-extensions, navigation, XAML, Navigation.Request, Navigation.Data, attached-properties, declarative-navigation, no-code-behind, data-binding, ListView, Button, event-binding, auto-binding, DataViewMap, ViewMap, RouteMap, element-binding, selection-navigation, back-navigation, route-based-navigation]
+---
+
+# Navigate Between Pages using Navigation Attached Properties in XAML
+
+## Purpose
+Demonstrates declarative navigation from XAML using `Navigation.Request` and `Navigation.Data` attached properties, enabling navigation without code-behind.
+
+## Prerequisites
+
+* Add `Navigation` support in your app's .csproj file:
+
+    ```diff
+    <UnoFeatures>
+        Material;
+        Extensions;
+    +   Navigation;
+        Toolkit;
+        MVUX;
+    </UnoFeatures>
+    ```
+
+* Import the Navigation namespace in XAML:
+
+    ```xml
+    <Page xmlns:uen="using:Uno.Extensions.Navigation.UI">
+    ```
+
+## Navigate Forward using Navigation.Request
+
+Use `Navigation.Request` attached property to trigger navigation declaratively.
+
+* Create a destination page `SamplePage.xaml` and ViewModel:
+
+    ```csharp
+    public class SampleViewModel
+    {
+        public SampleViewModel(INavigator navigator)
+        {
+            _navigator = navigator;
+        }
+
+        private readonly INavigator _navigator;
+    }
+    ```
+
+* Add a Button with `Navigation.Request` in `MainPage.xaml`:
+
+    ```xml
+    <Button Content="Go to SamplePage"
+            uen:Navigation.Request="Sample" />
+    ```
+
+* **Navigation.Request** — Attached property that specifies the route to navigate to
+* Route string `"Sample"` maps to the registered route for `SamplePage`
+* No Click event handler needed in code-behind
+
+### Auto-Attached Events
+
+`Navigation.Request` automatically attaches to appropriate events based on control type:
+
+* **Button** — Attaches to `Click` event
+* **ListView** — Attaches to `SelectionChanged` event
+* **Border/Image/TextBlock** — Attaches to `Tapped` event
+
+### Navigate Back using Navigation.Request
+
+* Use `"-"` as the route to navigate back:
+
+    ```xml
+    <Button Content="Go Back"
+            uen:Navigation.Request="-" />
+    ```
+
+* **"-" route** — Special route that navigates to the previous page
+
+## Register Routes for Navigation
```

Comment:
This should be its own page, with something titled like "Registering navigation routes and associating views to their view models